src/lib/frontend/components/ProductCardShadow.svelte

```diff
@@ -1 +1 @@
-<div class="w-full h-[20.5rem] sm:h-[19.5rem] md:h-[16.1rem] xl:h-[14rem] bg-slate-50 animate-pulse"></div>
\ No newline at end of file
+<div class="h-[326px] p-4 rounded-10 shadow flex-col justify-start items-start gap-[25px] inline-flex w-full bg-slate-200 animate-pulse"></div>
```

Comment:
Do we need a component for a shadow? Can't this be part of the base component for product cards?